src/components/containers/TransformAccordion.js

```diff
@@ -32,39 +36,64 @@ class TransformAccordion extends Component {
       );
     }
 
+    const transformBy =
+      container.transforms &&
+      container.transforms.map(tr => {
+        let retValue = '';
```

Comment:
This could be `const` with a ternary operator

Reply:
retValue? I don't see how - it can have 3 values, including ''

Reply:
ok

Reply:
what is `retValue`, returnValue?
would you mind giving it a name that says more what it's used for, 
I guess this one's for the label, so `columnLabel`?